src/object/player.cpp

```diff
@@ -2324,8 +2325,13 @@ Player::collision_solid(const CollisionHit& hit)
 
   // crushed?
   if (hit.crush) {
-    if (hit.left || hit.right) {
-      kill(true);
+    if ((hit.left || hit.right)) {
+      if (m_ignore_sideways_crush) {
+        m_ignore_sideways_crush = false;
+      }
+      else {
+        kill(true);
+      }
```

Comment:
Couldn't we make it so that despite of the side where Tux was hit from, `kill(false)` is performed? Is there a reason for sideways crushing to instantly kill Tux with `kill(true)`?

If we do that, `m_ignore_sideways_crush` wouldn't be needed.